cumulusci/tasks/salesforce/tests/test_sourcetracking.py

```diff
@@ -256,4 +257,7 @@ def test_run_task__null_revnum(self):
     def test_freeze(self):
         task = create_task(SnapshotChanges)
         steps = task.freeze(None)
-        self.assertEqual([], steps)
+        assert steps == []
+
+
+flake8 = (create_task_fixture,)
```

Comment:
One way around this is to define fixtures in conftest.py so that pytest finds them without needing to be imported.

Reply:
We have a few create_task, _make_task etc. thingees floating around and I wasn't confident enough to promote this to being the global everyone-should-use-this one until I'd tried it in a few more contexts.

Reply:
Yeah I definitely want to try to consolidate those, but am also not necessarily asking you to do that right now.